automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsTestGroupAnnotations.java

```diff
@@ -17,11 +17,6 @@
 @Target({ElementType.METHOD, ElementType.TYPE})
 @interface BillingsTest {}
 
-@TagAnnotation
-@Retention(RetentionPolicy.RUNTIME)
-@Target({ElementType.METHOD, ElementType.TYPE})
-@interface DataRepoSnapshotsTest {}
-
```

Comment:
removing `DataRepoSnapshotsTest` is unrelated to this PR, but it was unused and I figured I'd clean it up while I was in there.